mobile-app/app/screens/WalletNavigator/screens/CreateWallet/CreateWalletGuidelines.tsx

```diff
@@ -1,131 +1,119 @@
+      <View style={tailwind('flex flex-row justify-center')}>
+        <Image
+          style={{ width: 200, height: 136 }}
+          source={isLight ? LightNewWallet : DarkNewWallet}
+        />
+      </View>
+      <ThemedTextV2
+        style={tailwind('mt-7 px-3 text-base font-normal-v2 text-center')}
       >
-        <ThemedText
-          dark={tailwind('text-darkprimary-500')}
-          light={tailwind('text-primary-500')}
-          style={tailwind('font-medium text-sm')}
-        >
-          {translate('screens/Guidelines', 'Learn more about recovery words')}
-        </ThemedText>
-      </TouchableOpacity>
-
-      {
-        guidelines.map((g, i) => (
+        {translate('screens/Guidelines', 'You will be shown 24 recovery words on the next screen. Keep your 24-word recovery safe as it will allow you to recover access to the wallet.')}
+      </ThemedTextV2>
+      <LearnMoreCTA onPress={() => navigation.navigate('RecoveryWordsFaq')} testId='recovery_words_faq' />
+      <View style={tailwind('px-6 mt-12')}>
+        {guidelines.map((g, i) => (
           <View
             key={i}
-            style={tailwind('flex-row items-center my-3')}
+            style={tailwind('flex flex-row items-start my-3')}
           >
-            <ThemedIcon
-              iconType='MaterialIcons'
-              name={g.icon}
-              size={32}
-            />
-
-            <View style={tailwind('flex-col flex-auto ml-6')}>
-              <ThemedText style={tailwind('font-medium')}>
-                {translate('screens/Guidelines', g.title)}
-              </ThemedText>
-
-              <ThemedText
-                dark={tailwind('text-gray-400')}
-                light={tailwind('text-gray-500')}
-                numberOfLines={4}
-                style={tailwind('text-sm')}
-              >
-                {translate('screens/Guidelines', g.subtitle)}
-              </ThemedText>
-            </View>
+            <ThemedViewV2
+              light={tailwind('bg-mono-light-v2-700')}
+              dark={tailwind('bg-mono-dark-v2-700')}
+              style={tailwind('flex h-6 w-6 flex-row items-center justify-center rounded-full mt-1')}
+            >
+              <ThemedIcon
+                light={tailwind('text-mono-light-v2-100')}
+                dark={tailwind('text-mono-dark-v2-100')}
+                iconType='Feather'
+                name={g.icon}
+                size={14}
+              />
+            </ThemedViewV2>
+            <ThemedTextV2
+              light={tailwind('text-mono-light-v2-700')}
+              dark={tailwind('text-mono-dark-v2-700')}
+              style={tailwind('flex-1 text-xs ml-4 font-normal-v2')}
+            >
+              {translate('screens/Guidelines', g.title)}
+            </ThemedTextV2>
           </View>
-        ))
-      }
-
-      <View style={tailwind('flex-row items-center my-4')}>
-        <Switch
-          onValueChange={toggleSwitch}
-          testID='guidelines_switch'
```

Comment:
can replace this one since it's already removed
<img width="463" alt="image" src="https://user-images.githubusercontent.com/13292662/179892325-80c87c6b-af25-4f65-bd3b-b87319335ee8.png">

Reply:
opsie. done and pushed